### ISSUE TITLE
When I test this client lib, and find that in the Android, the string send to the server lost the last character. 

### ISSUE THREAD

**Issue by wqmeng** [mateusvicente100/bird-socket-client#3]
Such as 

Send 'Hello server',  On the server side, only get 'Hello serve' .  Not sure what the matter.

**Comment by mateusvicente100** [comment]
Hello @wqmeng. I didn't use the client on FMX yet, just on VCL. I don't know what about the compatibility of Indy Socket on FMX, but I will do a test to see this, or, if you can send me a simple client aplication witih this error to me test.

**Comment by wqmeng** [comment]
Hello @mateusvicente100 ,  

I ported the demo in the client package to Android,  and still the same result.  The server side will lost the last character send from the client.

I have attached the test project,  to deploy it,  it will need the ssl support,  in the deployment panel, set up two libcrypto.so, libssl.so

local path is  ssl\,  remote path is  assets\internal\

Thank you.

![Snipaste_2021-03-01_21-41-35](https://user-images.githubusercontent.com/216289/109505030-28714000-7ad7-11eb-9c96-39ab77cd0232.png)

[TestBirdWebsocketClient.zip](https://github.com/mateusvicente100/bird-socket-client/files/6061965/TestBirdWebsocketClient.zip)
